examples/rds/v1beta1/instance.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     backupRetentionPeriod: 14
     instanceClass: db.t3.micro
     engine: postgres
-    engineVersion: "13.7"
+    engineVersion: "16.1"
```

Comment:
nit: If there's one, we can consider using substitutes like `latest` to prevent the maintenance overhead of having to update these versions. This is in violation of version pinning but here we would be _testing_ the resource against the latest available version so I believe that's fine.

Reply:
Unfortunately, we need to specify an explicit version for this field.